docs/source/man.rst

```diff
@@ -511,7 +511,7 @@ The ``qpc report details`` command generates a detailed report that contains the
 
 ``--json``
 
-  Displays the results of the report in JSON format. Mutually exclusive with the ``--csv`` option.
+  Returns the results of the report in Javascript Object Notation (JSON) format compressed into a tarball (tar.gz). Mutually exclusive with the ``--csv`` option.
```

Comment:
I don't think we want to externalize this to the end user.  We should unzip it for them and put it in the file they specified.